src/tools/cephfs/top/cephfs-top

```diff
@@ -573,9 +678,82 @@ class FSTop(object):
 
             stats_json = self.perf_stats_query()
             try:
-                if self.refresh_header(stats_json):
+                headline = "HELP: Selected Filesystem info"\
+                    " [Press 'q' to go back to home (all filesystem info) screen"\
+                    " and 'm' to select another filesystem]"
+                if self.refresh_header(stats_json, headline):
                     x_coord_map = self.refresh_top_line_and_build_coord()
-                    self.refresh_main_window(x_coord_map, stats_json)
+                    self.refresh_main_window(x_coord_map, stats_json, fs)
+                curses.halfdelay(self.refresh_interval_secs)
+                c = self.header.getch()
+                if c in [ord('m'), ord('q')]:
+                    self.set_option(c)
+                else:
+                    continue
+                self.exit_ev.wait(timeout=self.refresh_interval_secs)
+            except curses.error:
+                # this may happen when addstr the terminal window
+                # size changed, just retry it
+                pass
+
+    def run_all_display(self, fs_list):
+        while not self.exit_ev.is_set():
+            # use stdscr.clear() instead of clearing each window
+            # to avoid screen blinking.
+            self.stdscr.clear()
+            self.refresh_window_size()
+            if self.height <= 4 or self.width <= 2:
+                self.exit_ev.wait(timeout=self.refresh_interval_secs)
+                continue
+
+            # coordinate constants for windowing -- (height, width, y, x)
+            # NOTE: requires initscr() call before accessing COLS, LINES.
+            try:
+                HEADER_WINDOW_COORD = (4, self.width - 1, 0, 0)
+                self.header = curses.newwin(*HEADER_WINDOW_COORD)
+            except curses.error:
+                # this may happen when creating the sub windows the
+                # terminal window size changed, just retry it
+                continue
+
+            stats_json = self.perf_stats_query()
+            try:
+                if not fs_list:
+                    headline = "INFO: No Filesystem is available"\
+                        " [Press 'q' to quit]"
```

Comment:
Suggest filesystem vs Filesystem

Reply:
Removed all occurences of Filesystems with filesystem.